test/test_helper.js

```diff
@@ -1,5 +1,7 @@
 const {expect} = require('chai');
+const fs = require('fs');
 const {Namespace, DataElement, Concept, Group, Value, CodeFromValueSetValue, CodeFromAncestorValue, RefValue, OrValues, QuantifiedValue, PrimitiveIdentifier, Identifier} = require('../lib/models');
+const validator = require('jsonschema').Validator;
```

Comment:
Is `validator` used anywhere?  It looks to me like both changes in this file are unnecessary and can be backed out.